lib/main/scrapper.js

```diff
@@ -0,0 +1,23 @@
+var cheerio = require('cheerio');
+var http = require('http');
+var url = require('url');
+
+function Scrapper(uri, success_callback, error_callback){
+  if(uri.indexOf('http') < 0){
```

Comment:
I believe this check should be a bit modified. You want to add http:// at start if it is not included but you should allow http anywhere else in URL.
e.g. How code will behave for following:
www.http.com